sdk/core/core-client/src/base64.ts

```diff
@@ -30,3 +30,11 @@ export function encodeByteArray(value: Uint8Array): string {
 export function decodeString(value: string): Uint8Array {
   return Buffer.from(value, "base64");
 }
+
+/**
+ * Decodes a base64 string into a string.
+ * @param value - the base64 string to decode
```

Comment:
Shouldnt there be an `@internal` tag on this just like the other functions in this file?

Reply:
I will make a PR! (the good news is that this isn’t being exported by the package anyway)